libs/src/utils/index.js

```diff
@@ -0,0 +1,17 @@
+const apiSigning = require('./apiSigning.js')
+const captcha = require('./captcha.js')
+const estimateGas = require('./estimateGas.js')
+const multiProvider = require('./multiProvider.js')
+const promiseFight = require('./promiseFight.js')
+const signatures = require('./signatures.js')
+const uuid = require('./uuid.js')
+
+module.exports = {
```

Comment:
can we export these dynamically? that way, we don't have to modify this file each time we want to add files to utils dir

Reply:
for something that we're exporting outside libs don't think we should auto require everything in the dir - seems safer to explicitly export here. this is also not a folder that will change frequently, would be better to force devs to manually update this if making changes

i see we do it elsewhere but that is only for auto generated artifacts where you don't know the filenames, and those are also for internal use

also for reference - https://itnext.io/dont-dynamically-import-code-in-a-directory-838cedf9534a

Reply:
👍 sounds good!